src/styles/_variables.scss

```diff
@@ -110,9 +114,19 @@ $h4-font-size-article:        $font-size-article * 1.33;
 $h5-font-size-article:        $font-size-article;
 $h6-font-size-article:        $font-size-article;
 
+$border-color:                $light;
+$table-border-color:          $border-color;
+$input-border-color:          $border-color;
+
+$input-padding-x:             1.2rem;

```

Comment:
Think this is more in taste, but can't we have one rule for padding? instead of two for sizing?

Reply:
So, to keep the height of the input according to the visual guidelines we need to do either one of these:
1. Add different padding
Or
2. Set fix height on the input (a solution we had in CUI 3).
I prefer solution 1 though, feel like we should avoid fixed height on element.